content/programming-cookbook/visual-studio/code-analysis.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Code Analysis
+showMetadata: true
+editable: true
+showToc: true
+---
+
+## Code Analysis
+Reference: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/overview
+
+## How do we know our code has great coverage of quality standards?
+This content we'll use Code Analysis with SonarLint extension.
+About SonarLint https://www.sonarlint.org/visualstudio/
+
+Let's setup!
+
+## Steps
+1. Go to Extensions and install SonarLint (We use Visual Studio 2019)
+   Visual Studio will need to close and install
+![Code Analysis](images/code-analysis-1.png)
+2. Open project and go to Analyze => Run Code Analysis => On Solution
+![Code Analysis](images/code-analysis-2.png)
+3. When done.
+   Go to Error List tab.
+![Code Analysis](images/code-analysis-3.png)
```

Comment:
If you don't need an alternate text for an image, please use empty square bracket. ![](image-url)

Reply:
@aaronamm I'm done, Thank you mery much.